samples/dymaptic.GeoBlazor.Core.Sample.Shared/Pages/WCSLayers.razor

```diff
@@ -0,0 +1,124 @@
+@page "/wcslayers"
+@using dymaptic.GeoBlazor.Core.Components.Renderers.ColorRamps;
+
+<h3>WCS Layers</h3>
+
+<div class="links-div">
+    <a class="btn btn-secondary" target="_blank" href="https://developers.arcgis.com/javascript/latest/api-reference/esri-layers-WCSLayer.html">ArcGIS API for JavaScript</a>
+    <a class="btn btn-primary" target="_blank" href="something">NOAA Sea Surface Temperature Charts</a>
+</div>
+<Label>Add the sample WCS Layer URL to see a visualization of global sea surface temperature data with or without color rendering using the RasterStretch:</Label>
+<InputText @bind-Value="_wcsLayerUrl"></InputText>
```

Comment:
I don't know that we really need this input field. I doubt anyone is going to test their own layers in our samples page.

Reply:
removed.  I think for some samples it makes sense to have the input field, others really dont need it.